samples/OptionAnalyzer.Test/UnitTests.fs

```diff
@@ -0,0 +1,36 @@
+module OptionAnalyzer.Test
+
+#nowarn "57"
+
+open NUnit.Framework
+open FSharp.Compiler.Text
+open FSharp.Analyzers.SDK
+
+[<SetUp>]
+let Setup () = ()
+
+[<Test>]
+let ``warning is emitted`` () =
+
+    let source =
+        """
+module M
+
+let notUsed() =
+    let option : Option<int> = None
+    option.Value
+"""
+
+    let expectedMsg =
+        {
+            Code = "OV001"
+            Fixes = []
+            Message = "Option.Value shouldn't be used"
+            Range = Range.mkRange "A.fs" (Position.mkPos 6 4) (Position.mkPos 6 16)
+            Severity = Severity.Warning
+            Type = "Option.Value analyzer"
+        }
+
+    let msgs = TestHelpers.getAnalyzerMsgs source
```

Comment:
Would it not make sense API-wise if we pass in the analyzer we want to test?
It is a little hidden what analyzers this is going to load.
I could imagine unit tests that start to fail because of the introduction of a new analyzer.

Something like `let ctx = TestHelpers.createContext source` to be passed to the `Analyzer`.
TestHelpers could also contain some assertion helpers.